examples/hello-compute/main.rs

```diff
@@ -13,7 +13,7 @@ async fn run() {
             .collect()
     };
 
-    log::info!("Times: {:?}", execute_gpu(numbers).await);
+    println!("Times: {:?}", execute_gpu(numbers).await);
```

Comment:
Could we keep this as `log::info!` and mention how to use it in a comment (e.g. `RUST_LOG=info cargo run --example hello-compute`)? We don't have `println!` on the wasm32 target but `log::info!` can be used with a logger that works there

Reply:
Oops, sorry, I switched it to println when I was debugging a little. Will switch it back.